content/_admin/ip-whitelisting.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Egress IP Addresses & IP Whitelisting
+layout: article
+section: elastic.io Specific Settings
+description: "Document lists public IP addresses of our clusters to whitelist if required."
+order: 1
+category: support
+---
+
+**Summary**
```

Comment:
Probably better to have this as a H2 header. This is good for the SEO purposes.

Reply:
I don't follow: I don't think it makes sense to try to optimize the word "Summay" for SEO.

Reply:
@jhorbulyk 

**Summary** -> ## Summary